File: washee_box/data_models/Machine.py
class Machine:

    def __init__(self, machineID, name, machineType, startTime=None, endTime=None):
        self.id = machineID
        self.name = name
        self.machineType = machineType
        self.startTime = startTime
        self.endTime = endTime
    
    def lock(self):
        return
    
    def unlock(self):
        return

File: washee_app/lib/features/location/domain/entities/machine.dart
import 'package:equatable/equatable.dart';

// ignore: must_be_immutable
class Machine extends Equatable {
  final String machineID;
  final String name;
  final String machineType;
  DateTime? startTime;
  DateTime? endTime;
  bool activated;

  Machine({
    required this.machineID,
    required this.name,
    required this.machineType,
    this.startTime,
    this.endTime,
    this.activated = false
  });

  @override
  List<Object?> get props => [machineID, name, machineType, startTime, endTime, activated];
}
